lasagne/tests/layers/test_helper.py

```diff
@@ -4,7 +4,7 @@
 import theano
 
 
-class TestGetAllLayers:
+class TestGetAllLayers(object):
```

Comment:
I do it by default nowadays. I guess old-style classes are frowned upon? I don't know :) Don't really care either way, as long as all classes in the library itself are new-style.